src/Components/Books/book/actionButtons.jsx

```diff
@@ -1,15 +1,15 @@
 import { PropTypes } from 'prop-types';
 
-const Button = ({ title, handleEvent, className }) => (
-  <button type="button" className={className} onClick={{ handleEvent }}>
+const Button = ({ title, handleClick, className }) => (
+  <button type="button" className={className} onClick={handleClick}>
     {title}
   </button>
 );
 
```

Comment:
- [ ] The _Check status_ button is missing under the _categories_ page! The categories page should have a button, _Check status_, which should use the function `checkCategoriesStatus` (name may be different in your app) made in the component to dispatch the `underConstruction` action creator. Kindly ensure to improve upon this aspect.

**Your categories status page:**
![image](https://user-images.githubusercontent.com/12745474/202286494-1875de3c-1da4-401a-95e7-563aa5d806da.png)